libcasr/src/js.rs

```diff
@@ -0,0 +1,379 @@
+//! JS module implements `ParseStacktrace` and `Exception` traits for JS reports.
+use crate::error::{Error, Result};
+use crate::exception::Exception;
+use crate::execution_class::ExecutionClass;
+use crate::stacktrace::{ParseStacktrace, Stacktrace, StacktraceEntry};
+
+use regex::Regex;
+
+/// Structure provides an interface for parsing JS exception message.
+pub struct JsException;
+
+impl Exception for JsException {
+    fn parse_exception(stderr: &str) -> Option<ExecutionClass> {
```

Comment:
We need to parse exception from stream error instead particular line

Reply:
Done